save-deploy/README.md

```diff
@@ -72,6 +72,18 @@ Usually not the whole stack is required for development. Application logic is pe
 * To make things easier, add line `save.profile=dev` to `gradle.properties`. This will make project version `SNAPSHOT` instead of timestamp-based suffix and allow caching of gradle tasks.
 * Run `./gradlew deployLocal -Psave.profile=dev` to start the database and run three microservices (backend, preprocessor and orchestrator) with Docker Compose.
 
+## Local debugging
+You can run backend, orchestrator, preprocessor and frontend locally in IDE in debug mode.<br/>
+If you run on windows, dependency `save-agent` is omitted because of problems with linking in cross-compilation.<br/>
+To run on windows and you need to compile save-agent on wsl and put saveCliDistribFilepath to %USERPROFILE%\.gradle\gradle.properties <br/>
+For example: `saveCliDistribFilepath=\\home\\username\\projects\\save-cloud\\save-agent\\build\\libs\\save-agent-0.3.0-alpha.0.48+1c1fd41-distribution.jar` <br/>
```

Comment:
Shouldn't this path start with `\wsl$` too?

Reply:
fixed